Learn-CSS-Blog/index9.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    <title>Home | Learn Basic CSS by building a Blog</title>
+</head>
+<body>
+    <div id="main">
+        <header>
+            <nav></nav>
+        </header>
+    </div>
+</body>
+</html>
+
+<!-- 
+    Nest a `ul` element inside of the `<nav>` element and give it a class of `nav-links`.
```

Comment:
Since this is a basic CSS course, we could make the the example code more closely match what learners will write:

    Nest a `ul` element inside of the `<nav>` element and give it a class of `nav-links`.
    Here is an example: 
     ```
        <nav>
            <ul class="social-row"></ul>
        </nav>
    ```